cms/static/sass/_base-v2.scss

```diff
@@ -6,8 +6,8 @@ html {
 
 body {
   min-width: $fg-min-width;
-  background: $gray-l5;
-  color: $gray-d2;
+  background: $uxpl-grayscale-x-back;
```

Comment:
I had to lighten the background to work with the blue links in our course navbar. I'm using UXPL colors here.